netlify/edge-functions/serve-definitions.ts

```diff
@@ -1,4 +1,4 @@
-import type { Context } from "netlify:edge";
+import type { Context } from "https://edge-bootstrap.netlify.app/v1/index.ts";
```

Comment:
The only concern here is that this URL could change in the future... Not really convinced I think 🤔

Reply:
I guess there is no better option at this moment 🤔